articles/quickstart/webapp/rails/01-login.md

```diff
@@ -33,7 +33,20 @@ If you are using Windows, uncomment the `tzinfo-data` gem in the Gemfile.
 
 Create a file named `auth0.rb` under `config/initializers` and configure the **OmniAuth** middleware in it.
 
-${snippet(meta.snippets.setup)}
```

Comment:
This is the only place in the repo that this snippet was being used. If you are no longer going to use a snippet, then please:
- remove the "import" from the index file: https://github.com/auth0/docs/blob/master/articles/quickstart/webapp/rails/index.yml#L12
- remove the snippet file from the snippets folder: https://github.com/auth0/docs/blob/master/snippets/server-platforms/rails/setup.md


OR you could edit that snippet file and keep this reference in here 👍

Reply:
@lbalmaceda - Missed this, apologies. This has been updated.